helpers/utils.js

```diff
@@ -0,0 +1,15 @@
+const sanitizeFilename = require('sanitize-filename');
+
+function convertToFilename(string, options = { replacement: '-', maxLength: 255 }) {
```

Comment:
why did you take this direction? did you face some issues while testing changes?